### PR TITLE
fix moduleclass & add custom sanity_check_paths in gettext easyconfigs

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 37 out of 37 (37 easyconfigs in this PR)
node2073.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/25b1e4bdf661291a64c994bb6c9190f3 for a full test report.

File: easybuild/easyconfigs/g/gettext/gettext-0.19.8.eb [hunk 1]
```diff
@@ -23,4 +23,16 @@ dependencies = [('ncurses', '6.0')]
...
+    'files': ['bin/gettext', 'lib/libasprintf.a', 'lib/libasprintf.%s' % SHLIB_EXT,
+              'lib/libgettextpo.a', 'lib/libgettextpo.%s' % SHLIB_EXT],
+    'dirs': ['include'],
+}
+
+sanity_check_paths = {
+    'files': ['bin/gettext', 'lib/libasprintf.a', 'lib/libasprintf.%s' % SHLIB_EXT,
+              'lib/libgettextpo.a', 'lib/libgettextpo.%s' % SHLIB_EXT],
+    'dirs': ['include'],
+}
```
Comment (verdurin) [inline easybuild/easyconfigs/g/gettext/gettext-0.19.8.eb]: @boegel - cut and paste error? Assume they're not supposed to be duplicated...

Comment (boegel) [reply easybuild/easyconfigs/g/gettext/gettext-0.19.8.eb]: Indeed, thanks for the thorough review!

I blame it on Friday afternoon...

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**